cli/ballerina-launcher/src/main/java/org/ballerinalang/launcher/BLauncherCmd.java

```diff
@@ -41,79 +37,11 @@
 
     void printUsage(StringBuilder out);
 
-    void setParentCmdParser(JCommander parentCmdParser);
-
-    void setSelfCmdParser(JCommander selfCmdParser);
-
-    @Deprecated
-    static String getCommandUsageInfo(JCommander cmdParser, String commandName) {
-        return getCommandUsageInfo(commandName);
-    }
-
-    static void printCommandList(JCommander cmdParser, StringBuilder out) {
-        int longestNameLen = 0;
-        for (JCommander commander : cmdParser.getCommands().values()) {
-            BLauncherCmd cmd = (BLauncherCmd) commander.getObjects().get(0);
-            if (cmd.getName().equals(BallerinaCliCommands.DEFAULT) || cmd.getName().equals(BallerinaCliCommands.HELP)) {
-                continue;
-            }
-
-            int length = cmd.getName().length() + 2;
-            if (length > longestNameLen) {
-                longestNameLen = length;
-            }
-        }
-
-        for (JCommander commander : cmdParser.getCommands().values()) {
-            BLauncherCmd cmd = (BLauncherCmd) commander.getObjects().get(0);
-            if (cmd.getName().equals(BallerinaCliCommands.DEFAULT) || cmd.getName().equals(BallerinaCliCommands.HELP)) {
-                continue;
-            }
-
-            String cmdName = cmd.getName();
-            String cmdDesc = cmdParser.getCommandDescription(cmdName);
+    void setParentCmdParser(CommandLine parentCmdParser);
```

Comment:
As we are refactoring, shall we add proper java doc comments for the API methods here?